frontend/src/scenes/Apps/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import styles from './styles.module.scss';
-import { Form, FormControl } from 'react-bootstrap';
+import { Form, FormControl, InputGroup, Card } from 'react-bootstrap';
+import { Search } from 'react-bootstrap-icons';
```

Comment:
Use font-awesome icons instead of react-bootstrap-icons

Reply:
Fixing complete